app/javascript/dashboard/routes/dashboard/settings/inbox/components/AddCampaign.vue

```diff
@@ -0,0 +1,202 @@
+            {{ $t('CAMPAIGN.ADD.FORM.TITLE.LABEL') }}
+            <input
+              v-model.trim="title"
+              type="text"
+              :placeholder="$t('CAMPAIGN.ADD.FORM.TITLE.PLACEHOLDER')"
+              @input="$v.title.$touch"
+            />
+          </label>
+        </div>
+
+        <div class="medium-12 columns">
+          <label :class="{ error: $v.message.$error }">
+            {{ $t('CAMPAIGN.ADD.FORM.MESSAGE.LABEL') }}
+            <textarea
+              v-model.trim="message"
+              rows="5"
+              type="text"
+              :placeholder="$t('CAMPAIGN.ADD.FORM.MESSAGE.PLACEHOLDER')"
+              @input="$v.message.$touch"
+            />
+          </label>
+        </div>
+
+        <div class="medium-12 columns">
+          <label :class="{ error: $v.selectedAgent.$error }">
+            {{ $t('CAMPAIGN.ADD.FORM.SENT_BY.LABEL') }}
+            <select v-model="selectedAgent">
+              <option
+                v-for="agent in agentsList"
+                :key="agent.name"
+                :value="agent.name"
+              >
+                {{ agent.name }}
+              </option>
+            </select>
+            <span v-if="$v.selectedAgent.$error" class="message">
+              {{ $t('CAMPAIGN.ADD.FORM.SENT_BY.ERROR') }}
+            </span>
+          </label>
+        </div>
+
+        <div class="medium-12 columns">
+          <label :class="{ error: $v.endPoint.$error }">
+            {{ $t('CAMPAIGN.ADD.FORM.END_POINT.LABEL') }}
+            <input
+              v-model.trim="endPoint"
+              type="text"
+              :placeholder="$t('CAMPAIGN.ADD.FORM.END_POINT.PLACEHOLDER')"
+              @input="$v.endPoint.$touch"
+            />
+            <span v-if="$v.endPoint.$error" class="message">
+              {{ $t('CAMPAIGN.ADD.FORM.END_POINT.ERROR') }}
+            </span>
+          </label>
+        </div>
+        <div class="medium-12 columns">
+          <label :class="{ error: $v.timeOnPage.$error }">
+            {{ $t('CAMPAIGN.ADD.FORM.TIME_ON_PAGE.LABEL') }}
+            <input
+              v-model.trim="timeOnPage"
+              type="number"
+              :placeholder="$t('CAMPAIGN.ADD.FORM.TIME_ON_PAGE.PLACEHOLDER')"
+              @input="$v.timeOnPage.$touch"
+            />
+            <span v-if="$v.timeOnPage.$error" class="message">
+              {{ $t('CAMPAIGN.ADD.FORM.TIME_ON_PAGE.ERROR') }}
+            </span>
+          </label>
+        </div>
+
+        <div class="medium-12 columns">
+          <label>
+            <input
+              v-model="enabled"
+              type="checkbox"
+              value="enabled"
+              name="enabled"
+            />
+            {{ $t('CAMPAIGN.ADD.FORM.ENABLED') }}
+          </label>
+        </div>
+
+        <div class="modal-footer">
+          <div class="medium-12 columns">
+            <woot-submit-button
+              :disabled="
+                $v.message.$invalid ||
```

Comment:
Can we move this condition to a computed since this has more than 3 variables? Which would improve readability